hadoop-hdds/docs/content/concept/StorageContainerManager.zh.md

```diff
@@ -4,7 +4,7 @@ date: "2017-09-14"
 weight: 3
 menu: 
   main:
-     parent: 概念
+     parent: 架构
```

Comment:
The rename of the parent menu I think can be handled in the new issue because it also contains other items.